helper.php

```diff
@@ -125,6 +125,42 @@ public function getTokenEndpoint($service) {
         $service = strtolower($service);
         return $this->getConf($service.'-tokenurl');
     }
+
+    /**
+     * @param bool $string if true returns a nice string for output, otherwise returns array of strings
+     *
+     * @return array|string
+     */
+    public function getValidDomains($string = false) {
+        $validDomains = explode(',', trim($this->getConf('mailRestriction'), ','));
```

Comment:
the first part: done

the second part: if such a feature exists, then it is undocumented: https://www.dokuwiki.org/devel:configuration#classes